plugins/modules/layman.py

```diff
@@ -46,8 +47,8 @@
     type: str
   validate_certs:
     description:
-      - If V(false), SSL certificates will not be validated. This should only be set to V(false) when no other option exists. Prior to 1.9.3 the
-        code defaulted to V(false).
+      - If V(false), SSL certificates will not be validated. This should only be set to V(false) when no other option exists.
+        Prior to 1.9.3 the code defaulted to V(false).
```

Comment:
This refers to Ansible 1.9.3, which is no longer relevant :)
```suggestion
      - If V(false), SSL certificates will not be validated. This should only be set to V(false) when no other option exists.
```